src/sentry/models/monitorcheckin.py

```diff
@@ -51,6 +51,7 @@ class MonitorCheckIn(Model):
     guid = UUIDField(unique=True, auto_add=True)
     project_id = BoundedBigIntegerField(db_index=True)
     monitor = FlexibleForeignKey("sentry.Monitor")
+    monitor_environment = FlexibleForeignKey("sentry.MonitorEnvironment", null=True)
```

Comment:
still need to add migration. also, this is going to have to be nullable until check_monitor is updated

Reply:
Yeah, we'll need to backfill too right